libdevcore/JSON.h

```diff
@@ -23,22 +23,97 @@
 #pragma once
 
 #include <json/json.h>
+#include <sstream>
+#include <memory>
 
 namespace dev
 {
 
+namespace json
+{
+
+/// StreamWriterBuilder that can be constructed with specific settings
+class StreamWriterBuilder : public Json::StreamWriterBuilder {
```

Comment:
Please don't put a space in front of `:` (except in `x ? y : z`).

Reply:
Ok.